### PR TITLE
ADD: Tabs for events

### PR DESCRIPTION
**Pull request by kherkatary**
## ADD:
-  Tabs added for particular days
- Carousal of the event cards in Phone View

### Issues to resolve : 
- Responsiveness of the Bar under Active Tab

### CODE REVIEW

File: src/Components/EventTab/EventTab.module.scss [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      height: fit-content;
+      font-weight: 600;
+    }
+
+    span {
+      font-weight: 600;
+      background: var(--srijan4-gradient-10);
+      background-clip: text;
+      -webkit-text-fill-color: transparent;
+      -webkit-background-clip: text;
```
Comment (JOSU9435) [inline src/Components/EventTab/EventTab.module.scss]: use class or id selectors instead of HTML element selectors

**Review by JOSU9435 — CHANGES_REQUESTED** [review]
image

- increase the space between cards make them evenly spaced

image

-  properly align the bottom bar below "Day3" with the text and make it a bit thicker.

- Use a single Json for both EventTab and EventCarousal

File: src/Components/EventTab/EventTab.module.scss (hunk 1 above)
Comment (kherkatary) [reply src/Components/EventTab/EventTab.module.scss]: FIXED: suggested changes resolved
- spacing problem fixed
- data extracted from single json
- removing the use of html selectors

File: src/Components/EventCarousal/EventCarousal.jsx
Comment (JOSU9435) [inline src/Components/EventCarousal/EventCarousal.jsx:1]: remove commented code

File: src/Components/EventTab/EventTab.jsx [hunk 2]
```diff
@@ -1,7 +1,98 @@
...
+import EventCarousal from "../EventCarousal/EventCarousal";
 
 const EventTab = () => {
-  return <div className={styles.container}>.styles</div>;
+  const { day1, day2, day3 } = DayData;
+  const [dayId, setDay] = useState(day1);
+  const [barOne, setbarOne] = useState(styles.long);
+  const [barTwo, setbarTwo] = useState(styles.short);
+  const [barThree, setbarThree] = useState(styles.short);
+  const [activeBar, setActiveBar] = useState(styles.activeOne);
```
Comment (JOSU9435) [inline src/Components/EventTab/EventTab.jsx]: you should combine these different data variables in a single object state

variables are not in the proper naming convention like `setbarOne` should be `setBarOne`

**Review by JOSU9435 — CHANGES_REQUESTED** [review]
image

the bottom bar is still not correctly aligned with the text it looks ugly. Fix it.

**Review by JOSU9435 — COMMENTED** [review]
- Animation of tabs are not implemented

**Review by JOSU9435 — CHANGES_REQUESTED** [review]
- Animation of tabs is not implemented

**Review by JOSU9435 — APPROVED** [review]

**Pull request merged**